app.py

```diff
@@ -71,27 +106,48 @@ def process_image():
             output_path
             ])
 
+
 if __name__ == '__main__':
     global upload_directory
-    global results_img_directory
-    global image_colorizer
+    global results_img_directory, results_video_directory
+    global image_colorizer, video_colorizer
 
     upload_directory = '/data/upload/'
     create_directory(upload_directory)
 
     results_img_directory = '/data/result_images/'
     create_directory(results_img_directory)
 
+    results_video_directory = '/data/video/result/'
+    create_directory(results_video_directory)
+
     model_directory = '/data/models/'
     create_directory(model_directory)
     
-    artistic_model_url = 'https://www.dropbox.com/s/zkehq1uwahhbc2o/ColorizeArtistic_gen.pth?dl=0'
-    get_model_bin(artistic_model_url, os.path.join(model_directory, 'ColorizeArtistic_gen.pth'))
 
+    # could be either 
+    # BOTH for both Video and Image
+    # IMAGE for Image API Only
+    # VIDEO for Video API Only
+    # the purpose is to avoid
+
+    api_type = os.getenv('COLORIZER_API_TYPE', 'BOTH')
+
+    image_colorizer = None
+    video_colorizer = None
+    
+    if api_type == 'IMAGE' or api_type == 'BOTH':
+        artistic_model_url = 'http://storage.gra5.cloud.ovh.net/v1/AUTH_18b62333a540498882ff446ab602528b/pretrained-models/image/deoldify/ColorizeArtistic_gen.pth'
```

Comment:
What about DeOldify original models? I do believe that if you want to use OVH models you definitely should remove all this code from DeOldify, this API code does seems part of DeOldify.

Reply:
This is not an ovh model it's the same as the dropbox one except that dropbox download is really slow compared the object storage . I can revert it to be dropbox.

Reply:
Well, Dropbox can be slow, but you download only once right?

Also, depending on where is your storage you pay traffic usage for all downloads that you will get from this URL, are you sure you want that?